app/src/main/res/values/strings.xml

```diff
@@ -715,7 +727,7 @@
     <!-- Preference for exceptions tracking protection -->
     <string name="preference_exceptions">Exceptions</string>
     <!-- Description for the exceptions preference screen-->
-    <string name="preference_exceptions_description">You have disabled Content Blocking for these websites.</string>
+    <string name="preference_exceptions_description2">You have disabled content blocking for these websites.</string>
```

Comment:
No need to change string ID for this, as each locale handles capitalization differently than English. So, for capitalization changes in English, no need for a new string ID :) Otherwise localizers have to translate  again the entire strings. We usually only change string ID if the sentence really changes in context